zebra-state/src/service/check/anchors.rs

```diff
@@ -199,13 +396,13 @@ pub(crate) fn fetch_sprout_final_treestates(
 /// (which must be populated with all treestates pointed to in the `prepared` block;
 /// see [`fetch_sprout_final_treestates()`]); or in the interstitial
 /// treestates which are computed on the fly in this function.
-#[tracing::instrument(skip(sprout_final_treestates, block, transaction_hashes))]
-pub(crate) fn sprout_anchors_refer_to_treestates(
+#[tracing::instrument(skip(sprout_final_treestates, block))]
```

Comment:
There can be hundreds of transactions in a block.

```suggestion
#[tracing::instrument(skip(sprout_final_treestates, block, transaction_hashes))]
```